app/models/cohort.rb

```diff
@@ -12,26 +12,30 @@ class Cohort < ApplicationRecord
   scope :current, -> { where('open_date <= ? AND close_date >= ?', Date.today, Date.today) }
   scope :closed, -> { where('open_date > ? OR close_date < ?', Date.today, Date.today) }
 
+  def order_by_id_and_state
+    applications.order('state DESC, id')
```

Comment:
:+1: to pulling out a method here. This method doesn't need to be a class method?

Reply:
never mind! looks good.